rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.7.1-release-notes.adoc

```diff
@@ -0,0 +1,13 @@
+= RPA Builder 6.7.1 Release Notes
+
+RPA Builder is the on-premise application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+This version is internal only. All features and fixed issues for this version are implemented in version xref:rpa-home::release-notes/rpa-builder-6.7.2-release-notes.adoc[6.7.2].
```

Comment:
I understand that it's an internal version. But do you think you should add a date to it? If not, disregard the message.